components/brave_wallet_ui/constants/types.ts

```diff
@@ -353,6 +353,7 @@ export interface TokenInfo {
   decimals: number
   visible?: boolean
   icon?: string
+  logo?: string
```

Comment:
Why add `logo` when there's an `icon` field already?

Reply:
Logo was introduced in a separate PR and is required to be passed as an arg to the BraveWalletService api. So i needed to add it there. When we introduce token images, we will need to refactor everywhere that is using `icon` and replace it with `logo`.